src/index.html

```diff
@@ -1,51 +1,50 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Product</title>
 
     <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
-      crossorigin="use-credentials"
-    >
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="use-credentials">
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
 
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="logo">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="apple" class="nav__link">Apple</a>
```

Comment:
якщо малось на увазі id то потрібно додавати `#`
```suggestion
            <a href="#apple" class="nav__link">Apple</a>
```